webpack.config.js

```diff
@@ -109,13 +110,16 @@ const webpackConfig = {
 		splitChunks: {
 			chunks: 'all',
 			name: isDevelopment,
-			cacheGroups: {
+
+			maxAsyncRequests: 20,
+			maxInitialRequests: 5,
+			/*cacheGroups: {
```

Comment:
you can remove the `cacheGroups` now that you've isolated the cause right?

Reply:
yeah, but it's a bit fragile. we might want to leave in the tinymce cacheGroup just to force it.